### PR TITLE
Added Sorting Algorithms Course Playlist  By Amulya's Academy(changes-made)

### PR DESCRIPTION
**Pull request by rishikesh-b**
## What does this PR do?
This PR Added a Sorting Algorithms  Course By Amulya's Academy

### Description
This course is designed to provide students with a deep understanding of the core principles and applications of Sorting Algorithms. By the end of the course, students will be equipped with the knowledge to select, design, and implement Sorting Algorithms that are appropriate for real-world problems.

### Why is this valuable (or not)?
Yes, It's valuable as it provides free learning resource For Sorting Algorithms 
### How do we know it's really free?
It's Absolutely Available for Free On Amulya's Academy Channel

### For book lists, is it a book? For course lists, is it a course? etc.
It is a  free Course available on youtube

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
is there no "playlist"?

**Comment by rishikesh-b** [comment]
I have updated the original Playlist link Please check. Thank you

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -162,6 +162,7 @@
 * [Data Structures and Algorithms in Python Full Course for Beginners](https://www.youtube.com/watch?v=pkYVOmU3MgA) - Aakash N S (freeCodeCamp)
 * [Data Structures and Algorithms Specialization](https://www.coursera.org/specializations/data-structures-algorithms) - UC San Diego, HSE University
 * [Data Structures Easy to Advanced Course - Full Tutorial from a Google Engineer](https://www.youtube.com/watch?v=RBSGKlAvoiM) - William Fiset (freeCodeCamp)
+* [Data Structures Full Course](https://www.youtube.com/playlist?list=PL2_aWCzGMAwI3W_JlcBbtYTwiQSsOTa6P) - mycodeschool
```
Comment (eshellman) [inline courses/free-courses-en.md:165]:
```suggestion
* [Data Structures](https://www.youtube.com/playlist?list=PL2_aWCzGMAwI3W_JlcBbtYTwiQSsOTa6P) - mycodeschool
```
Don't invent titles! Use the title given by the resource.

**Comment by rishikesh-b** [comment]
@eshellman Sir Thanks for your Valuable Feedback. I have Made Changes according to the rules. Please Check it

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**